### PR TITLE
Update volumes.md

### PR DESCRIPTION
**Pull request by lparis**
Updating the link for the Kubernetes vSphere Cloud Provider and adding a more info link to the vSphere CSI documentation.

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -972,8 +972,7 @@ For more information about StorageOS, dynamic provisioning, and PersistentVolume
 ### vsphereVolume {#vspherevolume}
 
 {{< note >}}
-You must configure the Kubernetes vSphere Cloud Provider. For cloudprovider
-configuration, refer to the [vSphere Getting Started guide](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/).
+The following documentation assumes that the [Kubernetes vSphere Cloud Provider](https://github.com/kubernetes/cloud-provider-vsphere) is configured. See also the [VMware vSphere Container Storage Plug-in Documentation](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/index.html).
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/storage/volumes.md:975]:
```suggestion
The following documentation assumes that the [Kubernetes vSphere Cloud Provider](https://github.com/kubernetes/cloud-provider-vsphere) is configured. You may also refer the [VMware vSphere Container Storage Plug-in Documentation](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/index.html).
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Slight rephrasing

**Comment by divya-mohan0209** [comment]
@lparis : Would you please be able to make the requested changes?

**Comment by divya-mohan0209** [comment]
@lparis : Would you please be able to make the requested changes?

**Comment by Kartik494** [comment]
@lparis Could you please addressed the requested change and rebase occurring for continuing this.
Thanks

**Comment by zacharysarah** [comment]
@lparis 👋🏻 Please feel free to `/reopen` when you're ready to rebase and respond to feedback.

/close

**Pull request closed without merging**